src/tasks/push-components.js

```diff
@@ -58,7 +58,7 @@ const getDataFromPath = async (path) => {
 const createContentList = (content, key) => {
   if (content[key]) return content[key]
   else if (Array.isArray(content)) return [...content]
-  else return [content]
+  else return []
```

Comment:
I think it's better to change to a code similar to that,

```suggestion
  else return !isEmpty(content) ? [content] : []
```

and add a import in the top of the file, like this one:
```js
const { isEmpty } = require('lodash')
```

Reply:
Hi @MarkusWendorf you added the fix i mentioned in another file 🙃